BEIMA.Client/cypress/integration/Router.spec.js

```diff
@@ -4,20 +4,54 @@ describe("Router Doesn't Redirect On Valid Pages", () =>{
   it('Visits Root', () =>{
     cy.visit('http://localhost:3000')
     cy.url().should('eq','http://localhost:3000/')
+    cy.contains('Devices')
```

Comment:
cy.contains by itself seems like it might be an expensive operation based on the little bit I looked up on it. Should we be identifying the DOM object with cy.get().contains() rather than just using the contains method?

Reply:
Yeah that's a good idea, that way we can be more specific in our tests too